packages/tools/devtools/devtools-example/package.json

```diff
@@ -26,15 +26,14 @@
 		"prettier": "prettier --check . --cache --ignore-path ../../../../.prettierignore",
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../../.prettierignore",
 		"rebuild": "npm run clean && npm run build",
-		"start": "concurrently npm:watch:devtools && npm:watch:devtools-core && npm:watch:devtools-view && npm:start:client",
+		"start": "concurrently \"npm:watch:*\" npm:start:client",
```

Comment:
Just out of curiosity, do we not need the `watch` script inside `scripts` (just as we have `start:client` inside the `scripts`)?

Reply:
`concurrently` supports wildcard patters for the scripts you specify. I specified `watch:*`, which will result in all of our `watch:foo` scripts running in parallel.